pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/openshift/windows-machine-config-operator/version"
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/ssh"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
please import this as `core` (or `corev1` to align with the `metav1` import)

Reply:
please import this as core->core "k8s.io/api/core/v1"